Jelly/Classes/JellyPresentationController.swift

```diff
@@ -32,7 +32,8 @@ class JellyPresentationController : UIPresentationController {
         switch self.presentation.backgroundStyle {
         case .blur(let effectStyle):
             animateBlurView(effectStyle: effectStyle)
-        case .dimmed:
+        case .dimmed(let alpha):
+            dimmingView.backgroundColor = UIColor(white: 0.0, alpha: alpha)
```

Comment:
Setting the backgroundColor should not happen in _presentationTranstionWillBegin_. There is a dedicated function called _setupDimmedView_ which suits better lmao.